specification/ml/_types/TrainedModel.ts

```diff
@@ -59,15 +61,16 @@ export class TrainedModelStats {
 }
 
 export class TrainedModelDeploymentStats {
```

Comment:
For what it's worth, using the same order as the Elasticsearch code would generally be easier to review. With alphabetical ordering, some related fields are not close to each other. (I'm not asking you to change it.)

Reply:
I had the same thought, but then we have most spec fields in alphabetical order, so I went for coherence